PacketDotNet/TcpFields.cs

```diff
@@ -22,8 +22,8 @@
 
 namespace PacketDotNet
 {
-    /// <summary> IP protocol field encoding information.
-    ///
+    /// <summary> 
+    /// IP protocol field encoding information.
```

Comment:
lets drop this change from this commit, it should go into a separate commit as it isn't related to the packet dissection you are adding.